### ISSUE TITLE
Exception when downloading Eclipse project basic workspace

### ISSUE THREAD

**Issue by billreed63** [maqetta/maqetta#1454]
Eclipse project basic tests
Pull down the most recent Maqetta build

Steps
Start maqetta in mutli user mode. (repeat tests in single user mode)
Create a new project by going to New-->Project in the menu bar.
Type a name for the project, and click the "Enable Eclipse Support"
Create a new folder under WebContent by going to the New-->Folder in the title bar. Call the folder aaa
Create a new HTML file in the WebContent/aaa folder by going to New-->HTML File in the menu bar
Add some content to the new HTML file. Save and close
Download the entire workspace as a zip by clicking the "Download Workspace" icon on the File Explore shortcut bar.
Default settings are fine, click OK and save the file somewhere on the file system
Start Eclipse. Then go to your workspace and right click and select Import-->Import
In "General"" select "Existing Project into Workspace", Click the radio button next to "From Archive" then browse to find the zip file you just downloaded.
The Maqetta project should show up in the lists of projects. Select it and click OK

davinci/theme/ThemeUtils:4Uncaught TypeError: Cannot call method 'getName' of undefined

**Comment by JonFerraiolo** [comment]
Assigned to @childsb. Is this bug different than #1429? Who generates the message, Eclipse or Maqetta? If Maqetta, what action in Maqetta generates that message?

```
davinci/theme/ThemeUtils:4Uncaught TypeError: Cannot call method 'getName' of undefined
```

**Comment by billreed63** [comment]
It is maqetta, when I click the ok button on the download dialog.the dialog just hangs and when I look in the javascript console I see the exception. What is also interesting is on the download dialog the edit box for the html library has the word "undefined" in it

**Comment by childsb** [comment]
Can't connect to VPN and try on lab9, but this is working for me on local. i'll try again on monday.

**Comment by childsb** [comment]
i do see the "undefined" for HTML library, but I dont get an error. tested on lab9 with ff.

**Comment by JonFerraiolo** [comment]
I tried out the steps above in Win/FF and also did not see any errors when doing the download action.

**Comment by childsb** [comment]
I resolved the "undefined" folder with commit e57f533dce686620d639f23a9f6d590381a0caa5

i'm going to close this issue for now.  please reopen if you encounter/can reproduce.
